app/models/trainee.rb

```diff
@@ -128,6 +128,11 @@ class Trainee < ApplicationRecord
 
   scope :ordered_by_date, -> { order(updated_at: :desc) }
 
+  # Returns draft trainees first, then all trainees in any other state.
+  scope :ordered_by_drafts, -> { order(ordered_by_drafts_clause) }
+
+  paginates_per 30
```

Comment:
Default to 50?

Reply:
@defong I think 100 would be a good starting point. But in this case I think @AbigailMcP has dropped it lower so we can test in the review app. Will be raised before merging.

Reply:
I now have it so that it's set to 30 for review, QA and dev environments, but 100 in production. Can tweak if we need to. Maybe it should also be 100 in QA?